packages/tokens/non-fungible/src/storage.rs

```diff
@@ -373,3 +405,36 @@ pub fn check_spender_approval(e: &Env, spender: &Address, owner: &Address, token
         panic_with_error!(e, NonFungibleTokenError::InsufficientApproval);
     }
 }
+
+/// Low-level function for increasing the balance of `to`, without handling
+/// authorization.
+///
+/// # Arguments
+///
+/// * `e` - Access to the Soroban environment.
+/// * `to` - The address whose balance gets increased.
+/// * `amount` - The amount by which the balance gets increased.
+///
+/// # Errors
+///
+/// * [`NonFungibleTokenError::MathOverflow`] - If the balance of the `to` would
+///   overflow.
+pub fn increase_balance(e: &Env, to: &Address, amount: TokenId) {
+    let Some(balance) = balance(e, to).checked_add(amount) else {
+        panic_with_error!(e, NonFungibleTokenError::MathOverflow);
+    };
+    e.storage().persistent().set(&StorageKey::Balance(to.clone()), &balance);
+}
+
+/// Low-level function for decreasing the balance of `to`, without handling
+/// authorization. Balance cannot go negative.
+///
+/// # Arguments
+///
+/// * `e` - Access to the Soroban environment.
+/// * `to` - The address whose balance gets decreased.
+/// * `amount` - The amount by which the balance gets decreased.
+pub fn decrease_balance(e: &Env, from: &Address, amount: TokenId) {
+    let new_balance = balance(e, from).saturating_sub(amount);
```

Comment:
In case of this function to be called on its own, I suggest we replace `saturating_sub` with `checked_sub` and generate an error where the case it would fall below `0`